taxi/views.py

```diff
@@ -44,6 +65,26 @@
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_form.html"
```

Comment:
You don't need to override `template_name` here, `taxi/car_form.html` is the default template for CarCreateView.